### PR TITLE
feat(edit-page: VueJS example for headless rendering

### PR DESCRIPTION
**Pull request by fmontes**
### Proposed Changes
* VueJS example for headless
* No EMA ready yet

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### CODE REVIEW

File: ema/vuejs/src/main.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import App from './App.vue'
+import router from './router'
+
+import { registerGlobalComponents } from './components/global-components'
+
+const app = createApp(App)
+
+app.use(router)
+
+registerGlobalComponents(app)
```
Comment (zJaaal) [inline ema/vuejs/src/main.js:13]: should we provide this function?

**Review by zJaaal — APPROVED** [review]

**Review by manuelrojas — APPROVED** [review]

**Review by KevinDavilaDotCMS — APPROVED** [review]

File: ema/vuejs/.prettierrc.json
[the remark this answers is not in the document]
Comment (fmontes) [reply ema/vuejs/.prettierrc.json:1]: No, this is for this folder only

File: ema/vuejs/src/components/global-components.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+export const registerGlobalComponents = (app) => {
+  Object.entries(components).forEach(([name, component]) => {
+    app.component(name, component)
+  })
+
+  // Optionally attach the registry to the app instance for later reference
+  app.config.globalProperties.$registeredComponents = Object.keys(components)
+}
```
[the remark this answers is not in the document]
Comment (fmontes) [reply ema/vuejs/src/components/global-components.js:24]: I know

File: ema/vuejs/src/main.js (hunk 1 above)
Comment (fmontes) [reply ema/vuejs/src/main.js:13]: I'll guess we'll see when we make EMA out of this

**Pull request merged**